api/libretimeapi/views.py

```diff
@@ -0,0 +1,193 @@
+import os
+from django.conf import settings
+from django.http import FileResponse
+from django.shortcuts import get_object_or_404
+from rest_framework import status
```

Comment:
This got lost in refactor somehow...
```suggestion
from rest_framework import status, viewsets
```